### ISSUE TITLE
Prefix all classes with rz-

### ISSUE THREAD

**Issue by foucdeg** [angular-slider/angularjs-slider#272]
### Steps to reproduce
1. Create a slider with discrete steps : 
```javascript
var slider = {
      min: 3,
      max: 6,
      options: {
        floor: 0,
        ceil: 9,
        stepsArray: 'MLKJIHGFED'.split(''),
        showTicksValues: true
   }
};
```

### Expected behaviour
The slider elements should have class all prefixed with `rz-`, such as `rz-bubble`

### Actual behaviour
Individual ticks have a class called `tick`. 

Tick is a common name that could be used in other parts of a website so it's probably not best to use such a generic classname inside a component. In my case I was using a `tick` class for a form which shows a tick when a field is valid, I added CSS to that class which impacted my sliders.

**Comment by ValentinH** [comment]
:+1:

**Comment by ValentinH** [comment]
Fixed and release in 2.10.3.
